WalletWasabi.Fluent/ViewModels/Wallets/Buy/Workflows/WorkflowStep.cs

```diff
@@ -168,7 +168,9 @@ public virtual async Task<ChatMessage> EditMessageAsync(ChatMessage chatMessage)
 
 			if (StringValue(value) is { } userMessage)
 			{
-				newMessage = newMessage with { Text = userMessage };
+				var timestamp = GetTimestampFromMessage(chatMessage);
+
+				newMessage = newMessage with { Text = timestamp + userMessage };
```

Comment:
```suggestion
				newMessage = newMessage with { Text = GetTimestampFromMessage(chatMessage) + userMessage };
```

Since we don't do any checks on `timestamp`, maybe we could simplify this. WDYT?

Reply:
You're right, but I like my version slightly better for readability.